3p-ecosystem/src/main/java/com/google/homesampleapp/screens/commissionable/CommissionableFragment.kt

```diff
@@ -122,4 +145,100 @@ class CommissionableFragment : Fragment() {
       true
     }
   }
+
+  // -----------------------------------------------------------------------------------------------
+  // Composables
+
+  @Composable
+  private fun CommissionableRoute(commissionableViewModel: CommissionableViewModel) {
+    val beacons by commissionableViewModel.beaconsLiveData.observeAsState()
+    val beaconsList = beacons!!.toList()
```

Comment:
This may be better as `val beaconsList = beacons?.toList() ?: emptyList()` so you avoid the double bangs

Reply:
Good point. Thx.